lib/python/picongpu/plugins/data/png.py

```diff
@@ -9,9 +9,11 @@
 
 import numpy as np
 import os
-from scipy import misc
 import collections
 
```

Comment:
why did you leave an empty line between `import collections`and `from imageio import imread`?

Reply:
Simply forgot, is fixed.